pom.xml

```diff
@@ -2529,7 +2529,6 @@
                     <version>2.20.1</version>
                     <configuration>
                         <argLine>-Xms256m -Xmx512m</argLine>
-                        <reuseForks>false</reuseForks>
```

Comment:
it's not a good idea, we launch a lot of embedded stuff, It will probably be slower (if it works)

Reply:
Let's wait and see ;-)